projects/packages/admin-ui/src/class-jetpack-admin-menu.php

```diff
@@ -0,0 +1,83 @@
+<?php
```

Comment:
Is there a reason the file is called `class-jetpack-admin-menu.php`? I'm guessing we could drop the `jetpack` from the filename as the notice code sniffer suggests.

Reply:
yep